system.prop

```diff
@@ -34,6 +30,7 @@ ro.bluetooth.wipower=true
 camera.disable_zsl_mode=1
 persist.camera.HAL3.enabled=0
 persist.camera.cpp.duplication=false
+media.camera.ts.monotonic=0
```

Comment:
https://github.com/halogenOS/android_device_oneplus_oneplus2/commit/f1bdc02b37c90d5deb1c2b5dd4f614bfd39890db

This commit brings this. I just merge the camera tweaks in props into one.